### PR TITLE
Update desktop-changelog.rst

### PR DESCRIPTION
**Pull request by amyblais**
Essentially same as https://github.com/mattermost/desktop/pull/1031.

### CODE REVIEW

File: source/help/apps/desktop-changelog.rst [hunk 1]
```diff
@@ -1,6 +1,97 @@
...
+
+The Mattermost Desktop v4.3.0 release includes a change to how desktop notifications are sent from non-secure URLs (http://). Organizations using non-secure Mattermost Servers (http://) will need to update to Mattermost Server versions 5.16.0+, 5.15.1, 5.14.4 or 5.9.5 (ESR) to continue receiving desktop notifications when using Mattermost Desktop v4.3.0 or later.
+
+Improvements
+~~~~~~~~~~~~~~~
+
+All Platforms
+^^^^^^^^^^^^^
+
+- Added support for maintaining a user's online status while the desktop app is in the background as well as automatically setting the user to away/online if the screensaver goes on/off, the user locks/unlocks their computer or shuts down their computer.
```
Comment (esethna) [inline source/help/apps/desktop-changelog.rst]: Added support for maintaining a user's online status while the desktop app is in the background but the user is interacting with their computer

File: source/help/apps/desktop-changelog.rst [hunk 2]
```diff
@@ -1,6 +1,97 @@
...
+- Updated spellchecker dictionaries for English.
+- Added support for exposing Webview Developer Tools via View Menu.
+- Improved the styling of the session expiry mention badge in the tab bar.
+- Improved the wording of the bad certificate dialog to make it harder to just allow.
```
Comment (esethna) [inline source/help/apps/desktop-changelog.rst]: Improved the wording of the invalid certificate dialog

File: source/help/apps/desktop-changelog.rst [hunk 3]
```diff
@@ -1,6 +1,97 @@
...
+- Added support for Desktop menu bar accessibility improvements. This was fixed with the major version upgrade of Electron to v5.0.0.
```
Comment (esethna) [inline source/help/apps/desktop-changelog.rst]: Improved accessibility support for the menu bar items

File: source/help/apps/desktop-changelog.rst [hunk 4]
```diff
@@ -1,6 +1,97 @@
...
+
+Mac
+^^^^^^^^^^^^^
+
+- Added a flag to enable MacOS dark mode title bar.
+
+Architectural Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Major version upgrade of Electron to v5.0.0. Electron is the underlying technology used to build the Desktop apps.
```
Comment (esethna) [inline source/help/apps/desktop-changelog.rst]: We only need to state the latest upgrade to v5

File: source/help/apps/desktop-changelog.rst [hunk 5]
```diff
@@ -1,6 +1,97 @@
...
+- Major version upgrade of Electron to v4.2.4. Electron is the underlying technology used to build the Desktop apps.
+- Major version upgrade of Electron to v3.0.9. Electron is the underlying technology used to build the Desktop apps.
+
+Bug Fixes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+All Platforms
+^^^^^^^^^^^^^
+
+- Fixed various desktop app notification issues.
```
Comment (esethna) [inline source/help/apps/desktop-changelog.rst]: Propose order by impact

**Comment by esethna** [comment]
Looks good with the minor comments above

**Comment by amyblais** [comment]
Also should check links in https://docs.mattermost.com/install/desktop-msi-gpo.html.

**Comment by amyblais** [comment]
@deanwhillier Please help check if v4.3 download links are correct here:
 - https://github.com/mattermost/docs/pull/3049/files#diff-dcc9ba198b59be56a71e3af640d71201
 - https://docs.mattermost.com/install/desktop-msi-gpo.html

File: source/help/apps/desktop-changelog.rst [hunk 6]
```diff
@@ -1,6 +1,96 @@
...
+- Fixed an issue where clicking the tooltip button dismissed the tooltip.
+
+Mac
+^^^^^^^^^^^^^
+
+- Fixed an issue where using the red Close button to close the window caused a blank screen when the window was maximized.
+- Fixed an issue where ``Cmd + Option + Shift + v`` and ``Cmd + Shift + v`` didn't work on MacOS desktop app. This bug was fixed with the major version upgrade of Electron to v5.0.0.
+- Fixed an issue where the timezones were incorrect in OSX High Sierra. This bug was fixed with the major version upgrade of Electron to v4.0.0.
+
+Known Issues
```
Comment (deanwhillier) [inline source/help/apps/desktop-changelog.rst]: @esethna proposed moving this higher and included some tweaks to the changelog on the Desktop repo, should we apply here as well?

File: source/install/desktop.rst [hunk 7]
```diff
@@ -44,8 +43,6 @@ The following Group Policies are available:
 +----------------------------+-----------------------------------------------------------------------------+----------------------+
 | Group Policy               | Description                                                                 | Required Version     |
 +============================+=============================================================================+======================+
-| Enable Auto Updater        | If disabled, the app auto-updater is disabled.                              | 4.3 or later         |
```
Comment (deanwhillier) [inline source/install/desktop.rst]: Removed this as the feature was removed for v4.3.0

File: source/install/desktop.rst [hunk 8]
```diff
@@ -54,11 +51,11 @@ The following Group Policies are available:
 macOS 10.9+
 --------------------------------------------------
 
-1. Download `latest version of the Mattermost desktop app <https://releases.mattermost.com/desktop/4.2.3/mattermost-desktop-4.2.3-mac.zip>`__.
+1. Download `latest version of the Mattermost desktop app <https://releases.mattermost.com/desktop/4.3.0/mattermost-desktop-4.3.0-mac.dmg>`__.
```
Comment (deanwhillier) [inline source/install/desktop.rst]: Changed this to instructions for downloading and installing using the .dmg as the Github release doesn't list the .zip file. Can add that back in instead if desired!

**Comment by deanwhillier** [comment]
I submitted an update to Homebrew repo for Mattermost Desktop v4.3.0: https://github.com/Homebrew/homebrew-cask/pull/71050

File: source/help/apps/desktop-changelog.rst (hunk 6 above)
Comment (amyblais) [reply source/help/apps/desktop-changelog.rst]: Made the updates.

**Comment by deanwhillier** [comment]
@amyblais, https://docs.mattermost.com/install/desktop-msi-gpo.html needs a bit more of an update than just updating link and versions. I will add this to my list of todos and try to get it done tomorrow.

**Pull request merged**